modules/osdk-cli-reference-run.adoc

```diff
@@ -7,54 +7,36 @@
 
 The `operator-sdk run` command provides options that can launch the Operator in various environments.
 
-.`run` arguments
+== packagemanifests
+
+`run packagemanifests` subcommand deploys an Operator's package manifests with OLM. The command argument must be set to a valid package manifest root directory, for example, ``<project-root>/packagemanifests`.
+
+.`packagemanifests` arguments
 [options="header",cols="1,3"]
 |===
 |Arguments |Description
 
+|`--include` (string)
+|The file path to Kubernetes resource manifests, for example, `Role` object, Subscription. These supplement or override defaults generated by `run` or `cleanup`.
```

Comment:
"Subscription" with an initial cap--and specifically without backticks and followed by "resource" or "object" is eyecatching.  Please make this a general reference, such as "subscription", or an API object reference, as covered in the [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting).

Additionally, the part after the first comma is a sentence fragment.  My sugg:

"The file path to the Kubernetes resource manifests, such as `Role` objects and `Subscription` objects."

Or...
"The file path to the Kubernetes resource manifests. `Role` objects and `Subscription` objects are examples of resources in manifests."

You don't have to accept my sugg, but please revise to avoid the fragment.